README.md

```diff
@@ -10,93 +10,168 @@ providing QUnit-specific wrappers around the helpers contained in
-setResolver(Ember.DefaultResolver.create({namespace: App}));
+module('Visiting the homepage', {
+  beforeEach: function() {
+    App = startApp();
+  },
+  afterEach: function() {
+    Ember.run(App, App.destroy);
+  }
+});
 
-// otherwise something like:
-import Resolver from './path/to/resolver';
-import {setResolver} from 'ember-qunit';
-setResolver(Resolver.create());
+test('The signup form works', function(assert) {
+  assert.expect(2);
+  
+  visit('/');
+  
+  fillIn('.form>input', 'foo@bar.com');
+  click('.submit');
+  
+  andThen(function() {
+    assert.equal(currentPath(), 'results');
+    assert.equal($('.results>li').length, 2);
+  });
+});
 ```
 
-### Simple example:
+### Component Unit Tests
+
+[Ember Guide](http://guides.emberjs.com/v1.13.0/testing/testing-components/)
 
 ```js
-// tell ember qunit what you are testing, it will find it from the
-// resolver
-moduleForComponent('x-foo', 'XFooComponent');
+moduleForComponent('x-foo', 'XFooComponent', { 
+  unit: true,
+  needs: ['helper:pluralize-string']
+});
 
 // run a test
 test('it renders', function(assert) {
-  assert.expect(2);
+  assert.expect(3);
 
   // creates the component instance
-  var component = this.subject();
+  var subject = this.subject();
   assert.equal(component.state, 'preRender');
 
   // render the component on the page
   this.render();
   assert.equal(component.state, 'inDOM');
+  assert.equal(this.$('.foo').text(), 'bar');
 });
 ```
 
-### Complex example
+Unit tests are currently the default mode for component tests (this will eventually change to integration tests).  To flag a test as a unit test, either specify `unit: true` or include `needs: []` in the callbacks object.
+
+Unit tests have the advantage of giving you direct access to the component instance so you can test its internals.  Unit tests have the following features:
+
+- You have access to the component instance through `this.subject()`.
+- If you want to render the componenet's template, call either `this.render()` or `this.$()`.
+- Testing the component's template is through `this.$()`.
+- You are required to specify any dependencies other than the component's template in the `needs: []` option.  This includes helpers, services, partials, and any other componenets (with their templates) that are referenced.
+- Unit tests do not call most of the Ember lifecycle hooks.  `didInsertElement` and `willDestroyElement` will be called, but the remaining hooks introduced in Ember 1.13.x will not be.
+- There is no outer context for the component so testing things such as actions will require directly stubbing the actions on the component.
+
+### Component Integration Tests
 
 ```js
-// a more complex example taken from ic-tabs
-moduleForComponent('ic-tabs', 'TabsComponent', {
-
-  // specify the other units that are required for this test
-  needs: [
-    'component:ic-tab',
-    'component:ic-tab-panel',
-    'component:ic-tab-list'
-  ]
+moduleForComponent('x-foo', 'XFooComponent', {
```

Comment:
The second argument is not needed here.  `moduleForComponent('x-foo', { });` is perfect (since the description here is just repeating the component name from the first argument).